packages/conf-clang-format/conf-clang-format.1/opam

```diff
@@ -0,0 +1,19 @@
+opam-version: "2.0"
+maintainer: "https://github.com/ocaml/opam-repository/issues"
+homepage: "https://clang.llvm.org/docs/ClangFormat.html"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+authors: "The Clang Team"
+license: "NCSA"
+build: ["clang-format" "--version"]
+depexts: [
+  ["clang-format"] {os-family = "debian"}
+  ["clang-tools-extra"] {os-family = "fedora"}
```

Comment:
```suggestion
  ["clang-tools-extra"] {os-distribution = "fedora"}
```

Reply:
Thanks. Maybe then this change is not needed, I was just trying to uniform with other conf packages. Interesting to see that the family name is actually fedora now.

Reply:
I think the "real" family name (as in the OCaml constructor) might be `Centos`, so it might make sense to use that instead. However, if I grep `os-family` in `opam-repository`, I think `fedora` appears more often than `centos` and `rhel`. In any case, if there is a will to normalize it, no problem, just tell me which one to use and I'll change it.